### DISCUSSION TITLE
shortcut issues on 11, print screen capture

### DISCUSSION THREAD

**Discussion by theeeno** [flameshot-org/flameshot#2320] [Q&A]
i used to have the shortcut for this setup on print screen where it would launch the app and give me the option to select certain areas. Now it just launches the app and takes a screenshot of the entire screen. Does anyone know the exact command for the short cut to capture certain areas with the cursor? need the select area to grab feature back as a shortcut.

**Comment by theeeno** [comment]
found out it i was on wayland that i was experiencing these issues. works perfectly fine on xorg.

**Comment by mmahmoudian** [comment]
It is more of a gnome Wayland thing. I don't think KDE Wayland or Sway users are experiencing such issues.
